content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,846 @@
+
+{{< video src="/images/learning-guide/tutorials/rendering/vegetation-bending-tutorial/detailbendingtree.mp4" autoplay="true" loop="true" width="100%" muted="true" info="The tree in the Editor with detail bending applied, moving the leaves slightly." >}}
+
+### Add main bending
+The leaves move now, but the tree doesn't sway yet. In this step, you will add main bending, the overall sway and movement that the whole tree experiences.
+
+1. Open `VegetationBending_Common.azsli`.
+
+1. Above your `ProcessBending` function, add a function to make the tree sway. Using the current position of the vertex (after it has been changed from the detail bending) and the bending determined by the wind, you can bend the tree as a whole.
+   
+   ```glsl
+   float3 MainBending(float3 objectSpacePosition, float4 bending)
+   {
+      float windX = bending.x;
+      float windY = bending.y;
+      float bendScale = bending.z;
+
+      // More bending occurs higher up on the object
+      float bendFactor = objectSpacePosition.z * bendScale;
+      bendFactor *= bendFactor; 
+
+      // Rescale the displaced vertex position with the original distance to the object's center
+      // to restrict vertex movement to minimize deformation artifacts
+      float len = length(objectSpacePosition); 
+      float3 newPos = objectSpacePosition;
+      newPos.xy += float2(windX, windY) * bendFactor;
+
+      return normalize(newPos) * len;
+   }
+   ```
+
+1. Call the `MainBending` function after the call to the `DetailBending` function, in the `ProcessBending` function.
+   
+   ```glsl
+   if (o_color_isBound)
+   {
+      // Overall wind
+      float4 currentBending = SetUpWindBending(currentTime, worldPosition);
+
+      // Detail bending
+      float3 currentOutPosition = DetailBending(position, normal, color, currentTime, worldPosition, currentBending.w);
+
+      currentOutPosition = MainBending(currentOutPosition, currentBending);
+
+      adjustedWorldPosition = mul(objectToWorld, float4(currentOutPosition, 1.0));
+   }
+   ```
+
+1. Open the **Editor**. You should see that your tree sways and the leaves still bend. If you don't, try increasing the wind properties in the Material Editor.
+
+Amazing, your tree now sways and reacts to wind! Try to place multiple trees and observe how the trees sway differently when close together versus farther away. Also, add some lighting to make the trees pop!
+
+{{< video src="/images/learning-guide/tutorials/rendering/vegetation-bending-tutorial/tree3.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Three trees in the Editor, all swaying at a different pace with detail bending." >}}
+
+### Add motion vectors
+Because your tree is moving, you should add a *motion vector pass* so you can choose to add motion blur or other effects. For example, we have a [Temporal Anti-Aliasing (TAA)](/docs/atom-guide/features/taa.md) feature which uses the motion vectors.
```

Comment:
```suggestion
Since your tree moves, you can add cool effects by using a *motion vector pass*. Motion vectors are used by effects such as motion blur and [Temporal Anti-Aliasing (TAA)](/docs/atom-guide/features/taa.md).
```